packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx

```diff
@@ -99,6 +117,7 @@ export function CarouselViewContainer({
             >
                 {returnPages(pages, shouldPageMoveOnPress, nextPage)}
             </AnimatedPagerView>
+            {renderPagination()}
```

Comment:
```suggestion
            {showPagination && (
                 <Pagination pages={pages} activeIndex={currentIndex} setPage={setPage} />
             )}
```

Same, let's just inline it :)